src/ngx_http_upsync_module.c

```diff
@@ -2391,14 +2395,13 @@ ngx_http_upsync_add_timers(ngx_cycle_t *cycle)
     if (upsync_server == NULL) {
         return NGX_OK;
     }
-    upscf = upsync_server->upscf;
 
     ngx_log_debug1(NGX_LOG_DEBUG_HTTP, cycle->log, 0,
-                   "upsync_add_timers: shm_name: %V", upsync_server->pc.name);
+        "upsync_add_timers: shm_name: %V", "upsync_add_timers");
 
     srandom(ngx_pid);
     for (i = 0; i < upsync_ctx->upstream_num; i++) {
-
+        upscf = upsync_server[i].upscf;
```

Comment:
the line is not logic relations with the lines below, moved to the line 2417 above(tmp = upscf->upsync_interval), that will be more better?
what do you think?